src/quick-lint-js/error.h

```diff
@@ -88,6 +88,13 @@
       ERROR(QLJS_TRANSLATABLE("'await' is only allowed in async functions"),   \
             await_operator))                                                   \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_await_followed_by_arrow_function, "E177",                          \
```

Comment:
Minor: Now E177 is taken. xD Try E178. (I can fix this when I merge if you want.)